Description: Rename psl library to gmt-psl to resolve conflict with libpsl.
Author: Bas Couwenberg <sebastic@debian.org>
Bug: http://gmt.soest.hawaii.edu/issues/723
Bug-Debian: https://bugs.debian.org/790524

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -308,7 +308,7 @@ endif (HAVE_ZLIB)
 # set the build version (VERSION) and the API version (SOVERSION)
 set_target_properties (pslib
 	PROPERTIES
-	OUTPUT_NAME psl
+	OUTPUT_NAME gmt-psl
 	PREFIX "${LIB_PREFIX}"
 	IMPORT_PREFIX "${LIB_PREFIX}"
 	VERSION ${GMT_LIB_VERSION}
